src/strsearch_enc_2.cpp

```diff
@@ -403,9 +408,9 @@ int main()
   vecInt foundloc(0);
   for (size_t i = 0; i < vecResult.size(); i++) {
     auto unpackedVal = vecResult[i]->GetPackedValue();
-    for (size_t j = 0; j< ringsize; j++) {
-      if (unpackedVal[j] == 0) {
-      auto loc = i + offset[j];
+    for (size_t jj = 0; jj< ringsize; jj++) {
```

Comment:
jj vs j, any particular reason for this?

Reply:
Shadowed a variable higher up. My compiler was giving me some issues